macros/apply-tags/apply_mps_to_tags.sql

```diff
@@ -16,7 +17,13 @@
 
     {% for item in column_tags %}
       {%- if get_masking_policy_for_tag(item.tag) %}
```

Comment:
Can you try to use namespace when calling macros? e.g. dbt_tags. get_masking_policy_for_tag